libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -225,37 +252,20 @@ export class PaperHandlerDriver implements PaperHandlerDriverInterface {
     // According to the manual, "REAL-TIME USB PROTOCOL FORMAT" supports transferring out optional data.
     // The prototype doesn't support it, so leave out support from this function until needed.
   ): Promise<Result<T, CoderError>> {
-    await this.realTimeLock.acquire();
-
-    const transferOutResult = await this.transferOutRealTime(requestId);
-    assert(transferOutResult.status === 'ok'); // TODO: handling
-
-    const transferInResult = await this.transferInRealTime();
-    this.realTimeLock.release();
-    assert(transferInResult.status === 'ok'); // TODO: handling
+    return await this.webDeviceMutex.withLock(async (webDevice) => {
```

Comment:
All real-time commands call `handleRealTimeExchange` ie. none call `transferOutRealTime` or `transferInRealTime` directly. Therefore it's cleanest to handle locking in this helper once rather than with boilerplate in each real-time command.